src/pages/Graph/GraphPage.tsx

```diff
@@ -82,12 +88,12 @@ type ReduxProps = {
     edgeLabelMode: EdgeLabelMode,
     showSecurity: boolean,
     showUnusedNodes: boolean,
-    node?: NodeParamsType
+    node?: NodeParamsType,
+    queryTime?: TimeInMilliseconds
```

Comment:
I was pretty sure query time was in milliseconds in our rest api, but turns out i was wrong. If you choose to use ms instead of seconds because of my previous comment, feel free to do as you prefer.

Reply:
I think I prefer that we still use seconds in the query parameters, meaning that we use seconds in our backend API.  But up to the point that we make the API call I think it's fine to consistently use milliseconds in the client code, because javascript works best with ms (Date.now(), Date.getTime(), etc).  The parameter above is still a js method param, so I have it as ms.  The conversion it done just when constructing the url params.